backtester/backtest/backtest.go

```diff
@@ -280,6 +279,12 @@ func (bt *BackTest) setupExchangeSettings(cfg *config.Config) (exchange.Exchange
 			MaximumTotal: cfg.CurrencySettings[i].SellSide.MaximumTotal,
 		}
 		sellRule.Validate()
+
+		limits, err := exch.GetOrderExecutionLimits(a, pair)
+		if err != nil {
+			return resp, err
+		}
```

Comment:
Yeah, I actually agree with what you are saying, the majority of 'in sample' amounts are not going to execute on exchanges at all. I am wondering if I should still set it in exchange settings and have a check before an order occurs to reduce the amount. I think that might be better because when we switch over to out-of-sample testing we can immediately conform to exchange settings.